tests/Unit/Domain/CoordinateMaps/Test_Rotation.cpp

```diff
@@ -108,41 +109,46 @@ void test_rotation_2() {
   test_serialization(half_pi_rotation_map);
 
   test_coordinate_map_argument_types(half_pi_rotation_map, xi1);
-  }
-  void test_rotation_3() {
-    test_rotation_3(CoordinateMaps::Rotation<3>(0.0, 0.0, 0.0),
-                    std::array<double, 3>{{1.0, 0.0, 0.0}},
-                    std::array<double, 3>{{0.0, 1.0, 0.0}},
-                    std::array<double, 3>{{0.0, 0.0, 1.0}});
-
-    test_rotation_3(CoordinateMaps::Rotation<3>(M_PI_2, 0.0, 0.0),
-                    std::array<double, 3>{{0.0, 1.0, 0.0}},
-                    std::array<double, 3>{{-1.0, 0.0, 0.0}},
-                    std::array<double, 3>{{0.0, 0.0, 1.0}});
-
-    test_rotation_3(CoordinateMaps::Rotation<3>(M_PI, 0.0, 0.0),
-                    std::array<double, 3>{{-1.0, 0.0, 0.0}},
-                    std::array<double, 3>{{0.0, -1.0, 0.0}},
-                    std::array<double, 3>{{0.0, 0.0, 1.0}});
-
-    test_rotation_3(CoordinateMaps::Rotation<3>(-M_PI_2, 0.0, 0.0),
-                    std::array<double, 3>{{0.0, -1.0, 0.0}},
-                    std::array<double, 3>{{1.0, 0.0, 0.0}},
-                    std::array<double, 3>{{0.0, 0.0, 1.0}});
-
-    test_rotation_3(CoordinateMaps::Rotation<3>(0.0, -M_PI_2, 0.0),
-                    std::array<double, 3>{{0.0, 0.0, 1.0}},
-                    std::array<double, 3>{{0.0, 1.0, 0.0}},
-                    std::array<double, 3>{{-1.0, 0.0, 0.0}});
-
-    test_rotation_3(CoordinateMaps::Rotation<3>(0.0, M_PI_2, 0.0),
-                    std::array<double, 3>{{0.0, 0.0, -1.0}},
-                    std::array<double, 3>{{0.0, 1.0, 0.0}},
-                    std::array<double, 3>{{1.0, 0.0, 0.0}});
-  }
+  check_if_map_is_identity(CoordinateMaps::Rotation<2>{0.0});
+  CHECK(not CoordinateMaps::Rotation<2>(M_PI_2).is_identity());
+}
```

Comment:
It looks like you got the formatting restored in this commit. It'll be a lot easier to use `git blame` if you make sure this formatting is correct in the first commit.